.github/workflows/build_and_test.yml

```diff
@@ -10,10 +10,15 @@ on:
     paths-ignore:
       - '**/*.md'
 
+env:
+  CI: 1
+  GCS_UPLOAD_PREFIX: fake
```

Comment:
curiosity - what does this do?

Reply:
Not completely sure. When I ran it without this var, one of the tests failed, so I just copied it from the OSD build and test workflow